### ISSUE TITLE
change the name of the repo?

### ISSUE THREAD

**Issue by zaidalyafeai** [ARBML/Bohour#2]
I like the name `aroudh`, `import aroudh as dh`, what do you think @MagedSaeed?

**Comment by MagedSaeed** [comment]
I agree that this name is more apt. However, I am afraid it might be confused a little with this package [https://github.com/kariminf/aruudy]. What do u think?

**Comment by zaidalyafeai** [comment]
What is the advantage of our tool compred to that?

**Comment by MagedSaeed** [comment]
Let me investigate that package more and respond.

**Comment by zaidalyafeai** [comment]
Can you upload the evaluation set you created for Aroud, I want to compare ours to that. Between, it seems it is not maintained as the latest changes are two years ago.

**Comment by MagedSaeed** [comment]
Sorry, I did it in another [issue](https://github.com/ARBML/Bohour/issues/3)

**Comment by MagedSaeed** [comment]
I checked [aruudy](https://github.com/kariminf/aruudy). There is a notable overlap. However, there are distinctions.

Commonalities:

- Arudi style [need to evaluate both approaches]
- basic bahr information with patterns
- Zehafs and Tafeelat

Differences:

- This work covers qafiyah analysis, theirs does not.
- This work covers sub-bahrs, I could not identify this in theirs.
- This work covers more special cases in bohour's patterns, I could not identify this in theirs.
- They have in-depth pre-processing steps, more than this work.
- Their work finds bait's bahr from its diacritized form, while this work needs some extension to provide this functionality.

In general, I think both projects are built for two different purposes, hence these differences. This work seems to give more focus on the theory of Aroud while their work seems to give more focus on the applied use of Aroud, especially in software engineering settings.

My analysis:

I think we stick to _bohour_ for any thing related to bohours analysis and, of course, qafiyah. We evaluate _aruudi_ for arudi style and see if their implementation is better than _bohour_. What do you think @zaidalyafeai ?

**Comment by zaidalyafeai** [comment]
Working on evaluation now.
